extensions/omni_txn/migrate/5_retry_update.sql

```diff
@@ -0,0 +1,21 @@
+drop procedure if exists retry(stmts text, max_attempts int, repeatable_read boolean,
+                               collect_backoff_values boolean, params record);
+
+
+create procedure retry(
+    stmts text, 
+    max_attempts int default 10, 
+    repeatable_read boolean default false, 
+    collect_backoff_values boolean default false, 
+    timeout interval default null,          -- Adding timeout parameter or other changes
```

Comment:
It is better to add new parameters at the end as a general practice to ensure that those using these functions without names, don't suddenly get a signature mismatch.